Library/Homebrew/yard/templates/default/docstring/html/internal.erb

```diff
@@ -0,0 +1,5 @@
+<p class="note private">
+  <strong>This <%= object.type %> is part of an internal API.</strong>
+  This <%= object.type %> may only be used internally in repositories owned by <a href="https://github.com/Homebrew">Homebrew</a>.
```

Comment:
Does this include formulae and casks? Formulae can be extracted to third party taps via `brew extract`.

Reply:
@Bo98 I'd personally say: no, this does not include them (for that reason). I think it'd be worth clarifying here though 👍🏻